addons/clickable-quote-authors/userscript.js

```diff
@@ -0,0 +1,18 @@
+export default async function ({ addon, console }) {
+  const authors = document.getElementsByClassName("bb-quote-author");
+
+  for (const author of authors) {
+    const authorName = author.textContent.match(/(?<=^)[\w-]{2,30}(?= wrote:$)/);
```

Comment:
Would it be worth making an API request to check if the user actually exists?

Reply:
What if someone quotes [the actual TOLORS user](https://scratch.mit.edu/users/TOLORS/)?

Reply:
Would make it inconsistent.